### ISSUE TITLE
Image Scanning for CVs

### ISSUE THREAD

**Issue by jtfogarty** [kubeflow/testing#846]
This is a reminder to all working groups to setup image scanning during their CI/CD process.  

### AutoML
@andreyvelich, @gaocegege, @johnugeorge

### Deployment
@PatrickXYS, @animeshsingh, @mameshini, @vpavlin, @yanniszark 

### Manifests
@PatrickXYS, @vkoukis, @yanniszark 

### Notebooks
@StefanoFioravanzo,@elikatsis, @kimwnasptd, @thesuperzapper, @yanniszark 

### Pipelines
@james-jwu, @kramachandran, @paveldournov 

### Serving
@animeshsingh, @cliveseldon, @ellistarn, @rakelkar, @yuzisun 

### Training
@gaocegege, @johnugeorge, @terrytangyuan,

**Comment by thesuperzapper** [comment]
@jtfogarty, you can also use the @kubeflow/wg-notebooks-leads to save time in tagging

**Comment by PatrickXYS** [comment]
@jtfogarty To better handle the requests, can you elaborate on what's your expected image scanning process? 

This can help WG folks understand the intention as well

**Comment by jtfogarty** [comment]
@PatrickXYS will there be an Amazon ECR for Kubeflow?  if so, can we simply enable scan on push and then publish reports?

**Comment by PatrickXYS** [comment]
@jtfogarty Yes public ECR engineering support is ongoing, and we look forward to having it onboard next month or so.

But scan on push and reports are not able on public ECR yet, so that might lag a bit.

**Comment by thesuperzapper** [comment]
@PatrickXYS have there been any updates on this for ECR image scanning?
